src/WP_CLI/LanguagePackUpgrader.php

```diff
@@ -47,7 +47,7 @@ public function generic_strings() {
 	 *                        existing local file, it will be returned untouched.
 	 * @return string|\WP_Error The full path to the downloaded package file, or a WP_Error object.
```

Comment:
Docblock needs to be adapted here.

Reply:
@schlessera Can you suggest proper docblock update?

Reply:
@wojsmol You can copy the one from https://core.trac.wordpress.org/changeset/45262#file1

Including the alignment :)

Reply:
@swissspidy copied